app/frontend/src/aquifers/components/View.vue

```diff
@@ -307,18 +314,23 @@ a {
 }
 
 .aquifer-information-list dt {
+  padding-right: 2rem;
   font-weight: bold;
 }
 
 .aquifer-information-list dd {
-  padding-left: 1rem;
+  padding-left: 2rem;
```

Comment:
Vuetify also has spacing helpers you can use (which seems to be heavily used up there) but could potentially get rid of all these css with margin-bottom: 0

Reply:
I checked the Bootstrap Vue [text spacing documentation](https://getbootstrap.com/docs/4.3/utilities/spacing/) and it shows that there is no way to get a padding of `2rem`. A class of `pr-4` = `1.5rem` and `pr-5` = `3rem`.